primitive/message.go

```diff
@@ -183,9 +183,12 @@ func (m *Message) WithTag(tags string) *Message {
 
 func (m *Message) WithKeys(keys []string) *Message {
 	var sb strings.Builder
```

Comment:
How about `m.WithProperty(PropertyKeys, strings.Join(keys, PropertyKeySeparator))` ?

Reply:
@gaofeihang ping~

Reply:
That's a good advice. I have modified the PR. Thanks!